test/spec/geo/leaflet/leaflet-tiled-layer-view.spec.js

```diff
@@ -2,70 +2,132 @@ var Backbone = require('backbone');
 var LeafletTiledLayerView = require('../../../../src/geo/leaflet/leaflet-tiled-layer-view');
 
 describe('leaflet-tiled-layer-view', function () {
-  beforeEach(function () {
-    this.layerModel = new Backbone.Model({
-      urlTemplate: 'http://whatever.com',
-      subdomains: 'a',
-      opacity: 1,
-      attribution: 'CARTO',
-      errorTileUrl: 'http://matall.in',
-      maxZoom: 10,
-      minZoom: 0,
-      tms: false
+  describe('does not support high resolution screens', function () {
+    beforeEach(function () {
+      this.layerModel = new Backbone.Model({
```

Comment:
I don't like to add stuff to the context because of the linter won't work instead I use local scoped variables, but is just a matter of personal taste